### ISSUE TITLE
Segmentation Network

### ISSUE THREAD

**Issue by DQDH** [jiwoon-ahn/psa#7]
The provided code don't contain the code of segmentation network, can you give the code? thanks a lot.

**Comment by stygian2a** [comment]
The code gives you labels for a segmentation network. You can use your own then!

**Comment by DQDH** [comment]
I want to reappear this paper by the given code, so it's important to get the code for segmentation network.

**Comment by jiwoon-ahn** [comment]
No need to make your own segmentation network. For the result of Ours-DeepLab, I just used [https://bitbucket.org/deeplab/deeplab-public/src] without changing any hyperparameters.

**Comment by DQDH** [comment]
Thanks a lot. But I want to reappear the result  of Ours-ResNet38, Can you tell me the code you used in the paper?

**Comment by jiwoon-ahn** [comment]
The original ResNet-38 code will do. [https://github.com/itijyou/ademxapp] The only difference is that I adopted Adam instead of SGD.

**Comment by DQDH** [comment]
Thanks. And is DeepLabv1 you used in experiment?

**Comment by jiwoon-ahn** [comment]
Yup. Specifically, it's DeepLab-CRF-LargeFOV.

**Comment by DQDH** [comment]
Thank you for your patience.

**Comment by mcever** [comment]
Hi, I am working to reproduce the results reported in your paper, so I am wanting to train ResNet-38 on the refined CAMs I have created using this repo. Do you have a command or any code for how to train ResNet-38? The ademxapp repository does not include specific instructions on how to train a voc model, though it appears as if their voc.py is capable of doing so. I am curious as to how you initialized the weights, what crop size you used, and what origin-size you used. Currently, I am trying something like the following:

```
(ademxapp) austin@vader:~/ssd/ademxapp$ python issegm/voc.py --gpus 0,1 --data-root data/VOCdevkit --output train_out --phase train --weights models/voc_rna-a1_cls21_s8_coco_ep-0001.params --split train --crop-size 448 --batch-images 10 --to-epoch 1000 --debug --origin-size 1024

```

**Comment by czzerone** [comment]
> Hi, I am working to reproduce the results reported in your paper, so I am wanting to train ResNet-38 on the refined CAMs I have created using this repo. Do you have a command or any code for how to train ResNet-38? The ademxapp repository does not include specific instructions on how to train a voc model, though it appears as if their voc.py is capable of doing so. I am curious as to how you initialized the weights, what crop size you used, and what origin-size you used. Currently, I am trying something like the following:
> 
> ```
> (ademxapp) austin@vader:~/ssd/ademxapp$ python issegm/voc.py --gpus 0,1 --data-root data/VOCdevkit --output train_out --phase train --weights models/voc_rna-a1_cls21_s8_coco_ep-0001.params --split train --crop-size 448 --batch-images 10 --to-epoch 1000 --debug --origin-size 1024
> ```

hello，do you have reproduced the result of the ResNet-38. I got some trouble when I reproduced the result of the ResNet-38.
